### ISSUE TITLE
How to define that always falls back to locale without region

### ISSUE THREAD

**Issue by luicm** [onesky/plugin-ios-ota#15]
Hi, 
It is happening that the app sometimes "falling back to locale without region" and others "falling back to development region language". This means that some times the app is in English when it should be in Chinese. I have check and the translations are correct, both in the .strings files and in OneSky service. 

is there a way to control this?

**Comment by bretdabaker** [comment]
The OTA should always fall back to "locale without region" first before falling back to "development region language", it could be due to the translation data is not downloaded in the app yet.

Can you please turn on debug messages and copy and paste the console log?

**Comment by luicm** [comment]
Hello, 
Our app is localised in 14 languages, I have checked and this issue is only happening in Chinese(Simplified) and Chinese(Traditional). Here is the log:

`2016-04-05 17:22:59.476 OneSkyOTAPlugin - checking for update.
2016-04-05 17:22:59.914 OneSkyOTAPlugin - localized string not found for key: QardioArm, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.914 OneSkyOTAPlugin - localized string not found for key: QardioArm, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.915 OneSkyOTAPlugin - localized string found for key: QardioArm, value: QardioArm, developmentLocale: en
2016-04-05 17:22:59.921 OneSkyOTAPlugin - localized string not found for key: QardioCore, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.921 OneSkyOTAPlugin - localized string not found for key: QardioCore, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.921 OneSkyOTAPlugin - localized string found for key: QardioCore, value: QardioCore, developmentLocale: en
2016-04-05 17:22:59.925 OneSkyOTAPlugin - localized string not found for key: Friends & Family, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.925 OneSkyOTAPlugin - localized string not found for key: Friends & Family, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.925 OneSkyOTAPlugin - localized string found for key: Friends & Family, value: Friends & Family, developmentLocale: en
2016-04-05 17:22:59.928 OneSkyOTAPlugin - localized string not found for key: Profile, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.928 OneSkyOTAPlugin - localized string not found for key: Profile, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.928 OneSkyOTAPlugin - localized string found for key: Profile, value: Profile, developmentLocale: en
2016-04-05 17:22:59.930 OneSkyOTAPlugin - localized string not found for key: Settings, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.930 OneSkyOTAPlugin - localized string not found for key: Settings, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.930 OneSkyOTAPlugin - localized string found for key: Settings, value: Settings, developmentLocale: en
2016-04-05 17:22:59.932 OneSkyOTAPlugin - localized string not found for key: Help, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.932 OneSkyOTAPlugin - localized string not found for key: Help, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.932 OneSkyOTAPlugin - localized string found for key: Help, value: Help, developmentLocale: en
2016-04-05 17:22:59.935 OneSkyOTAPlugin - localized string not found for key: Support, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.935 OneSkyOTAPlugin - localized string not found for key: Support, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.935 OneSkyOTAPlugin - localized string found for key: Support, value: Support, developmentLocale: en
2016-04-05 17:22:59.937 OneSkyOTAPlugin - localized string not found for key: Buy now, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.937 OneSkyOTAPlugin - localized string not found for key: Buy now, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.937 OneSkyOTAPlugin - localized string found for key: Buy now, value: Buy now, developmentLocale: en
2016-04-05 17:22:59.939 OneSkyOTAPlugin - localized string not found for key: Engineering mode, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.939 OneSkyOTAPlugin - localized string not found for key: Engineering mode, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.939 OneSkyOTAPlugin - localized string found for key: Engineering mode, value: Engineering mode, developmentLocale: en
2016-04-05 17:22:59.941 OneSkyOTAPlugin - localized string not found for key: Logout, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.941 OneSkyOTAPlugin - localized string not found for key: Logout, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.941 OneSkyOTAPlugin - localized string found for key: Logout, value: Logout, developmentLocale: en
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string not found for key: MEASUREMENT, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string not found for key: MEASUREMENT, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string found for key: MEASUREMENT, value: MEASUREMENT, developmentLocale: en
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string not found for key: HISTORY, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string not found for key: HISTORY, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string found for key: HISTORY, value: HISTORY, developmentLocale: en
2016-04-05 17:22:59.953 OneSkyOTAPlugin - localized string not found for key: REMINDERS, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:22:59.954 OneSkyOTAPlugin - localized string not found for key: REMINDERS, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:22:59.954 OneSkyOTAPlugin - localized string found for key: REMINDERS, value: REMINDERS, developmentLocale: en
2016-04-05 17:23:00.054 OneSkyOTAPlugin - localized string not found for key: Underweight, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.054 OneSkyOTAPlugin - localized string not found for key: Underweight, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.055 OneSkyOTAPlugin - localized string found for key: Underweight, value: Underweight, developmentLocale: en
2016-04-05 17:23:00.055 OneSkyOTAPlugin - localized string not found for key: Normal, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.055 OneSkyOTAPlugin - localized string not found for key: Normal, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.056 OneSkyOTAPlugin - localized string found for key: Normal, value: Normal, developmentLocale: en
2016-04-05 17:23:00.056 OneSkyOTAPlugin - localized string not found for key: OverWeight, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.056 OneSkyOTAPlugin - localized string not found for key: OverWeight, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.056 OneSkyOTAPlugin - localized string found for key: OverWeight, value: OverWeight, developmentLocale: en
2016-04-05 17:23:00.057 OneSkyOTAPlugin - localized string not found for key: Obese, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.057 OneSkyOTAPlugin - localized string not found for key: Obese, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.057 OneSkyOTAPlugin - localized string found for key: Obese, value: Obese, developmentLocale: en
2016-04-05 17:23:00.057 OneSkyOTAPlugin - localized string not found for key: bmi, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.058 OneSkyOTAPlugin - localized string not found for key: bmi, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.058 OneSkyOTAPlugin - localized string found for key: bmi, value: bmi, developmentLocale: en
2016-04-05 17:23:00.058 OneSkyOTAPlugin - localized string not found for key: Body Fat, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.059 OneSkyOTAPlugin - localized string not found for key: Body Fat, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.059 OneSkyOTAPlugin - localized string found for key: Body Fat, value: Body Fat, developmentLocale: en
2016-04-05 17:23:00.059 OneSkyOTAPlugin - localized string not found for key: Muscle, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.059 OneSkyOTAPlugin - localized string not found for key: Muscle, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.060 OneSkyOTAPlugin - localized string found for key: Muscle, value: Muscle, developmentLocale: en
2016-04-05 17:23:00.061 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.061 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.061 OneSkyOTAPlugin - localized string found for key: Add note ≫, value: Add note ≫, developmentLocale: en
2016-04-05 17:23:00.062 OneSkyOTAPlugin - localized string not found for key: Set Goal, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.062 OneSkyOTAPlugin - localized string not found for key: Set Goal, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.062 OneSkyOTAPlugin - localized string found for key: Set Goal, value: Set Goal, developmentLocale: en
2016-04-05 17:23:00.069 OneSkyOTAPlugin - localized string not found for key: Measurement Details, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.069 OneSkyOTAPlugin - localized string not found for key: Measurement Details, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.069 OneSkyOTAPlugin - localized string found for key: Measurement Details, value: Measurement Details, developmentLocale: en
2016-04-05 17:23:00.071 OneSkyOTAPlugin - localized string not found for key: Mode, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.072 OneSkyOTAPlugin - localized string not found for key: Mode, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.072 OneSkyOTAPlugin - localized string found for key: Mode, value: Mode, developmentLocale: en
2016-04-05 17:23:00.074 OneSkyOTAPlugin - localized string not found for key: Last Measurement, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.075 OneSkyOTAPlugin - localized string not found for key: Last Measurement, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.075 OneSkyOTAPlugin - localized string found for key: Last Measurement, value: Last Measurement, developmentLocale: en
2016-04-05 17:23:00.082 OneSkyOTAPlugin - localized string not found for key: GUEST, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.082 OneSkyOTAPlugin - localized string not found for key: GUEST, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.082 OneSkyOTAPlugin - localized string found for key: GUEST, value: GUEST, developmentLocale: en
2016-04-05 17:23:00.082 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.082 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.082 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.087 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.087 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.087 OneSkyOTAPlugin - localized string found for key: Add note ≫, value: Add note ≫, developmentLocale: en
2016-04-05 17:23:00.121 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.121 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.121 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.121 OneSkyOTAPlugin - localized string not found for key: last %d days, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.121 OneSkyOTAPlugin - localized string not found for key: last %d days, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.121 OneSkyOTAPlugin - localized string found for key: last %d days, value: last %d days, developmentLocale: en
2016-04-05 17:23:00.126 OneSkyOTAPlugin - localized string not found for key: Lose, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.126 OneSkyOTAPlugin - localized string not found for key: Lose, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.126 OneSkyOTAPlugin - localized string found for key: Lose, value: Lose, developmentLocale: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: Target, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: Target, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string found for key: Target, value: Target, developmentLocale: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: Weekly, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: Weekly, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string found for key: Weekly, value: Weekly, developmentLocale: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.127 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.128 OneSkyOTAPlugin - localized string not found for key: GUEST, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.128 OneSkyOTAPlugin - localized string not found for key: GUEST, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.128 OneSkyOTAPlugin - localized string found for key: GUEST, value: GUEST, developmentLocale: en
2016-04-05 17:23:00.128 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.129 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.129 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.129 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.129 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.129 OneSkyOTAPlugin - localized string found for key: Add note ≫, value: Add note ≫, developmentLocale: en
2016-04-05 17:23:00.131 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.131 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.131 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.131 OneSkyOTAPlugin - localized string not found for key: last %d days, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.131 OneSkyOTAPlugin - localized string not found for key: last %d days, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.131 OneSkyOTAPlugin - localized string found for key: last %d days, value: last %d days, developmentLocale: en
2016-04-05 17:23:00.133 OneSkyOTAPlugin - localized string not found for key: Lose, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.133 OneSkyOTAPlugin - localized string not found for key: Lose, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.133 OneSkyOTAPlugin - localized string found for key: Lose, value: Lose, developmentLocale: en
2016-04-05 17:23:00.133 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.133 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string not found for key: Target, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string not found for key: Target, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string found for key: Target, value: Target, developmentLocale: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string not found for key: Weekly, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string not found for key: Weekly, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.134 OneSkyOTAPlugin - localized string found for key: Weekly, value: Weekly, developmentLocale: en
2016-04-05 17:23:00.135 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.135 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.135 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.627 OneSkyOTAPlugin - localized string not found for key: GUEST, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.628 OneSkyOTAPlugin - localized string not found for key: GUEST, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.628 OneSkyOTAPlugin - localized string found for key: GUEST, value: GUEST, developmentLocale: en
2016-04-05 17:23:00.629 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.629 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.629 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.630 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.630 OneSkyOTAPlugin - localized string not found for key: Add note ≫, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.630 OneSkyOTAPlugin - localized string found for key: Add note ≫, value: Add note ≫, developmentLocale: en
2016-04-05 17:23:00.633 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.634 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.634 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.634 OneSkyOTAPlugin - localized string not found for key: last %d days, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.634 OneSkyOTAPlugin - localized string not found for key: last %d days, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.634 OneSkyOTAPlugin - localized string found for key: last %d days, value: last %d days, developmentLocale: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: Lose, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: Lose, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string found for key: Lose, value: Lose, developmentLocale: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: Target, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: Target, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string found for key: Target, value: Target, developmentLocale: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.637 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.638 OneSkyOTAPlugin - localized string not found for key: Weekly, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.638 OneSkyOTAPlugin - localized string not found for key: Weekly, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.638 OneSkyOTAPlugin - localized string found for key: Weekly, value: Weekly, developmentLocale: en
2016-04-05 17:23:00.638 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.638 OneSkyOTAPlugin - localized string not found for key: kg, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.638 OneSkyOTAPlugin - localized string found for key: kg, value: kg, developmentLocale: en
2016-04-05 17:23:00.707 OneSkyOTAPlugin - translations are up-to-date.
2016-04-05 17:23:00.719 OneSkyOTAPlugin - localized string not found for key: Take Your Weight Measurement, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.719 OneSkyOTAPlugin - localized string not found for key: Take Your Weight Measurement, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.719 OneSkyOTAPlugin - localized string found for key: Take Your Weight Measurement, value: Take Your Weight Measurement, developmentLocale: en
2016-04-05 17:23:00.719 OneSkyOTAPlugin - localized string not found for key: Open App, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:00.719 OneSkyOTAPlugin - localized string not found for key: Open App, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:00.720 OneSkyOTAPlugin - localized string found for key: Open App, value: Open App, developmentLocale: en
2016-04-05 17:23:02.394 OneSkyOTAPlugin - localized string not found for key: Take Your Weight Measurement, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:02.394 OneSkyOTAPlugin - localized string not found for key: Take Your Weight Measurement, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:02.394 OneSkyOTAPlugin - localized string found for key: Take Your Weight Measurement, value: Take Your Weight Measurement, developmentLocale: en
2016-04-05 17:23:02.394 OneSkyOTAPlugin - localized string not found for key: Take Your Weight Measurement, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:02.395 OneSkyOTAPlugin - localized string not found for key: Take Your Weight Measurement, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:02.395 OneSkyOTAPlugin - localized string found for key: Take Your Weight Measurement, value: Take Your Weight Measurement, developmentLocale: en
2016-04-05 17:23:02.396 OneSkyOTAPlugin - localized string not found for key: Open App, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:02.396 OneSkyOTAPlugin - localized string not found for key: Open App, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:02.396 OneSkyOTAPlugin - localized string found for key: Open App, value: Open App, developmentLocale: en
2016-04-05 17:23:02.396 OneSkyOTAPlugin - localized string not found for key: Open App, locale: zh-Hans-NL, falling back to locale without region: zh
2016-04-05 17:23:02.397 OneSkyOTAPlugin - localized string not found for key: Open App, locale: zh-Hans-NL, falling back to development region language: en
2016-04-05 17:23:02.399 ---[3356:1149592] OneSkyOTAPlugin - localized string found for key: Open App, value: Open App, developmentLocale: en
`

Thanks for your help

**Comment by luicm** [comment]
I think I may find the issue. Since Chinese made differences in between: Chinese(Simplified) and Chinese(Traditional), when it fall back into the regio it take out the post fix and instad of looking for `zh-Hans` or `zh-Hant` is trying to looking for `zh` and obviously it didn't find it. 

here you can see the logs for french and for portuguese: 
```
2016-04-05 17:40:26.859 OneSkyOTAPlugin - localized string not found for key: QardioArm, locale: fr-NL, falling back to locale without region: fr

2016-04-05 17:40:26.860 OneSkyOTAPlugin - localized string found for key: QardioArm, value: QardioArm, locale without region: fr

2016-04-05 17:40:26.866 OneSkyOTAPlugin - localized string not found for key: QardioCore, locale: fr-NL, falling back to locale without region: fr

2016-04-05 17:40:26.866 OneSkyOTAPlugin - localized string found for key: QardioCore, value: QardioCore, locale without region: fr
```

```
2016-04-05 17:44:46.831 OneSkyOTAPlugin - checking for update.

2016-04-05 17:44:47.235 OneSkyOTAPlugin - localized string found for key: QardioArm, value: QardioArm, locale: pt-PT

2016-04-05 17:44:47.241 OneSkyOTAPlugin - localized string found for key: QardioCore, value: QardioCore, locale: pt-PT

2016-04-05 17:44:47.244 OneSkyOTAPlugin - localized string found for key: Friends & Family, value: Amigos & Família, locale: pt-PT

```

Please let me know if this is something that i can fix myself or it needs an update of the OTA plugin. 
Kind regards
Luisa

**Comment by luicm** [comment]
Hello, do you have any update?

**Comment by bretdabaker** [comment]
We will update the OTA plugin to fix the problem, hopefully by tomorrow.

**Comment by luicm** [comment]
great, thank you very much

**Comment by bretdabaker** [comment]
Fixed in [v0.9.9](https://github.com/onesky/plugin-ios-ota/releases/tag/0.9.9)
